### DISCUSSION TITLE
nach ota firmwareupdate des esp32 sind die Funkmoduleinstellunen des nrf24 nicht mehr konfiguriert in open dtu

### DISCUSSION THREAD

**Discussion by rythm1955** [tbnobody/OpenDTU#1412] [Q&A]
was kann ich wie wo tun 
bin auf dem Gebiet absoluter Laie

**Comment by rythm1955** [comment]
gelöst mit neuer pin_mapping.json

**Reply by marcoxxx1978** [reply]
hi, 
wo find ich die pin_mapping.json ?
bin absoluter anfänger

**Reply by Solarteur** [reply]
Beispiel mit weiterem Link gibt es hier:

https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles.md

**Reply by marcoxxx1978** [reply]
hm .. wie find ich raus was für ein pin mapping drauf ist und wie erstellt man ein wie spielt man das ein ?
fragen über fragen .... vielleicht gibt es schon eine dokumentation für dummis oder jemand hat so viel gedult mir das zu erklären ?
Danke schon man im vorraus
